app/helpers/items_helper.rb

```diff
@@ -50,6 +52,11 @@ def category_nav(categories, current_category = nil)
     end
   end
 
+  def status_hint(item)
```

Comment:
Originally I just did this in the template, but erb_lint didn't like me using `html_safe` there. Standard is okay with it though 🤷